### DISCUSSION TITLE
PMD User Survey

### DISCUSSION THREAD

**Discussion by adangel** [pmd/pmd#3814] [General]
We as PMD (<https://pmd.github.io>) maintainers want to learn more about our users, so that we can make PMD even better.

You can help in giving us some insights, how you actually use PMD.

This survey is anonymous. We might however publish a summary of the results publicly.

Survey URL to share: <https://forms.gle/4d8r1a1RDzfixHDc7>

Thank you for participating!

**Comment by rsoesemann** [comment]
@adangel can I publicly share this survey yet?

**Reply by adangel** [reply]
Yes

**Reply by rsoesemann** [reply]
Tweeted https://twitter.com/rsoesemann/status/1498754009913610241

**Comment by oowekyala** [comment]
I created #3829 to also have this show in the Issues tab

**Comment by adangel** [comment]
The survey is finished now.
Thank you for all your answers and feedback!
Read here about the results: https://pmd.github.io/2022/07/28/PMD-User-Survey/
